docs/rfcs/003-consolidate-core-packages.md

```diff
@@ -0,0 +1,338 @@
+| **[welcome]** | 0 | 0 | 0 | 11/21/17 |
+| **[whitespace]** | 31 | 6 | 0 | 5/30/18 |
+| **[wrap-guide]** | 3 | 4 | 0 | 11/27/17 |
+
+#### Packages to be Consolidated Later
+
+The following packages will not be consolidated until the stated reasons can be resolved or we decide on a consolidation strategy for them:
+
+| Package | Open Issues | Open PRs | Outside Maintainers | Last Updated | Reason |
+|---------|-------------|----------|---------------------|--------------|-------|
+| **[find-and-replace]** | 219 | 17 | 0 | 6/4/18 | Too many open PRs |
+| **[fuzzy-finder]** | 89 | 22 | 0 | 5/17/18 | Too many open PRs |
+| **[language-c]** | 53 | 15 | 0 | 7/10/18 | Too many open PRs |
+| **[language-go]** | 12 | 2 | **1** | 6/18/18 | Package maintainer, possibly inactive? |
+| **[language-java]** | 8 | 2 | **1** | 6/11/18 | Package maintainer |
+| **[language-javascript]** | 66 | 12 | 0 | 7/6/18 | Too many open PRs |
+| **[language-perl]** | 17 | 1 | **1** | 10/30/17 | Package maintainer, possibly inactive? |
+| **[markdown-preview]** | 139 | 12 | 0 | 1/8/18 | Too many open PRs |
+| **[settings-view]** | 137 | 18 | 0 | 5/17/18 | Too many open PRs |
+| **[snippets]** | 57 | 4 | **1** | 4/17/18 | Package maintainer |
+| **[solarized-dark-syntax]** | 8 | 3 | **1** | 5/27/18 | Package maintainer |
+| **[solarized-light-syntax]** | 2 | 3 | **1** | 5/27/18 | Package maintainer |
+| **[spell-check]** | 68 | 14 | **1** | 5/25/18 | Too many open PRs, package maintainer |
+| **[symbols-view]** | 86 | 13 | 0 | 12/10/17 | Too many open PRs |
+| **[tree-view]** | 210 | 36 | 0 | 3/21/18 | Too many open PRs |
+
+#### Packages to Never Consolidate
+
+These packages will not be consolidated for the following reasons:
+
+| Package | Open Issues | Open PRs | Outside Maintainers | Last Updated | Reason |
+|---------|-------------|----------|---------------------|--------------|-------|
+| **[autocomplete-atom-api]** |  |  |  |  | Blocks contribution from Facebook |
+| **[autocomplete-css]** |  |  |  |  | Same as above |
+| **[autocomplete-html]** |  |  |  |  | Same as above |
+| **[autocomplete-plus]** |  |  |  |  | Same as above |
+| **[autocomplete-snippets]** |  |  |  |  | Same as above |
+| **[github]** |  |  |  |  | Independent project |
+
+### Consolidation Process
+
+To consolidate a single core package repository back into `atom/atom`, the following steps will be taken:
+
+1. All open pull requests on the package's repository must either be closed or merged before consolidation can proceed
+1. The package repository's code in `master` will be copied over to Atom's `packages` folder in a subfolder bearing that package's name.
+1. Atom's `package.json` file will be updated to change the package's `packageDependencies` entry to reference its local path with the following syntax: `"tree-view": "file:./packages/tree-view"`
+1. A test build will be created to manually verify the package loads and works correctly at first glance and also that package specs pass
```

Comment:
Can you help me understand what this means? Does this mean locally running `script/build --install`, locally running the package's specs, and locally performing some manual sanity testing to verify the package's functionality?

Reply:
That's correct, I'll update the language to be more clear!